src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int color = new Random().nextInt(Color.values().length);
```

Comment:
Instead of calling `Color.values()` multiple times, which creates a new array each time, store it in a local variable and use that variable to access the array.